src/classes/CW_AudienceGroupTriggerHandlerTest.cls

```diff
@@ -0,0 +1,236 @@
+/**
+* File Name   :    CW_AudienceGroupTriggerHandlerTest.cls
+* Description :    D-106803o
+*                 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 24/04/2014 Alex  Panka		Initial
+ */
+ 
+@isTest
+public with sharing class CW_AudienceGroupTriggerHandlerTest {
+	
+	static testMethod void ValidationRuleTest() {
```

Comment:
* I suggest to annotate test methods with `@IsTest` for better readability.
* Methods should use the `lowerCamelCase` naming convention. I suggest rename accordingly.